### PR TITLE
[BLOCKED] drag to reorder server cards

### PR DESCRIPTION
**Pull request by daniellacosse**
BLOCKER: currently working on a [Storybook integration](https://github.com/Jigsaw-Code/outline-client/pull/1237) to get to the bottom of the draggableJS polyfill issues

---

https://user-images.githubusercontent.com/3759828/144281095-8550a97e-2ce6-4597-bd71-d0b3855b9b41.mov

<details>
<summary>TODOs</summary>

- [ ] test on mobile (SortableJS should take care of this)
- [x] make sure package-lock.json was updated properly
- [ ] how to handle magic parameters?
- [ ] update server config on change

</details>

### CODE REVIEW

File: src/tsconfig.json [hunk 1]
```diff
@@ -6,6 +6,7 @@
     "noImplicitAny": true,
     "noImplicitThis": true,
     "moduleResolution": "Node",
+    "esModuleInterop": true,
```
Comment (daniellacosse) [inline src/tsconfig.json:9]: with the addition of SortableJS, typescript would not build without this (i also tried `import * as Sortable`)

Is there any reason we had this set to false?

File: src/www/ui_components/server_list.ts [hunk 2]
```diff
@@ -43,19 +50,21 @@ export class ServerList extends PolymerElement {
...
-          expanded="[[hasSingleServer]]"
-          localize="[[localize]]" 
-          root-path="[[rootPath]]" 
-          server-address="[[item.address]]" 
-          server-id="[[item.id]]"
-          server-name="[[item.name]]" 
-          is-outline-server="[[item.isOutlineServer]]"
-        ></server-card>
-      </template>
+      <div id="sortableContainer">
```
Comment (daniellacosse) [inline src/www/ui_components/server_list.ts:53]: digging into the source code, SortableJS seems to not support dragging across multiple nested web components, so I opted to create a container here

**Comment by daniellacosse** [comment]
Went ahead and rebased on our recent changes. After removing all computed properties from the server card (this crashes the app when DraggableJS tries to clone the element), this is the behavior we see on android and ios, respectively:

https://user-images.githubusercontent.com/3759828/159305107-8c5b4a82-82fd-497b-9bbc-b54921434b50.mov

This behavior largely remains unchanged from before we fixed the selector logic, so the problem here is deeper. I'm going to proceed with setting up the Storybook to try to isolate these issues.

_(Also, the text isn't showing I suspect because something's happening w/ `localize` when draggableJS clones the element.)_

Why do we pass localize down as opposed to it being a service we can import? @fortuna

**Review by fortuna — COMMENTED** [review]
localize is part of the state so it knows when to update the strings when the language changes.

**Comment by daniellacosse** [comment]
> localize is part of the state so it knows when to update the strings when the language changes.

What if we had the localized strings passed down, and when the language changes, the strings would get regenerated?

**Pull request closed without merging**